src/router.js

```diff
@@ -8,16 +8,21 @@ module.exports = function createRouter (routes, options) {
   options = options || {}
 
   options.mode = options.mode || 'history'
-  options.interceptLinks = options.interceptLinks !== false
   routes = flatten(routes)
 
   var onTransition
   var history
   var qs = options.qs || defaultQs
-  var unintercept = options.interceptLinks && links.intercept(shouldIntercept, onClick)
+  var unintercept
 
-  function shouldIntercept (url) {
```

Comment:
So the problem I have is that I use a catch all '*' route for a 404 not found page, meaning it will always match a route